.github/workflows/tests.yml

```diff
@@ -54,6 +54,9 @@ jobs:
       - name: Unit tests
         run: yarn jest --ci --maxWorkers=1
         env:
+          CLOUDINARY_CLOUD_NAME: keystone-cloudinary
+          CLOUDINARY_KEY: 758252673115372
+          CLOUDINARY_SECRET: ${{ secrets.CLOUDINARY_SECRET }}
```

Comment:
Hmmm, we're going to need to conditionally run the tests or something given the secret won't be available on PRs from forks.

Reply:
@mitchellhamilton: That's a valid point. 👍